lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -726,6 +726,11 @@ abstract class ExchangeViewModelBase extends WalletChangeListenerViewModel with
 
               /// return after the first successful trade
               return;
+            } on SwapXyzProviderException catch (e) {
+              tradeState = TradeIsCreatedFailure(
+                title: S.current.trade_not_created,
+                error: e.message);
+              return;
```

Comment:
was this for debugging? else why do we return if swapxyz failed? why not continue trying with others?